app/models/payroll/claim_csv_row.rb

```diff
@@ -0,0 +1,110 @@
+# frozen_string_literal: true
+
+require "delegate"
+require "csv"
+require "excel_utils"
+
+module Payroll
+  class ClaimCsvRow < SimpleDelegator
+    DATE_FORMAT = "%m/%d/%Y"
+
+    VALUES = [
```

Comment:
Why are these constant values put in this `VALUES` array constant?

Reply:
No reason in particular, just felt a bit strange having the them as top level class constants. Felt more readable scoping them under `VALUES` to show they are values in the CSV row. I might be overthinking it :)

Reply:
Ah right. I think it's fine to have them as high-level constants. I think it's stranger and potentially more confusing to have a constant with a list of values that doesn't get used anywhere. A comment-as-heading at the top could be used to indicating what these constants represent if you feel it would be useful to give an indication of that.